wizard/WizardHome.qml

```diff
@@ -0,0 +1,246 @@
+                imageIcon: "../images/create-wallet.png"
+
+                onMenuClicked: {
+                    wizardController.restart();
+                    wizardController.createWallet();
+                    wizardStateView.state = "wizardCreateWallet1"
+                }
+            }
+
+            Rectangle {
+                Layout.preferredHeight: 1
+                Layout.topMargin: 3 * scaleRatio
+                Layout.bottomMargin: 3 * scaleRatio
+                Layout.fillWidth: true
+                color: MoneroComponents.Style.dividerColor
+                opacity: MoneroComponents.Style.dividerOpacity
+            }
+
+            WizardMenuItem {
+                headerText: qsTr("Create a new wallet from hardware") + translationManager.emptyString
+                bodyText: qsTr("Connect your hardware wallet to create a new Monero wallet.") + translationManager.emptyString
+                imageIcon: "../images/restore-wallet-from-hardware.png"
+
+                onMenuClicked: {
+                    wizardController.restart();
+                    wizardStateView.state = "wizardCreateDevice1"
+                }
+            }
+
+            Rectangle {
+                Layout.preferredHeight: 1
+                Layout.topMargin: 3 * scaleRatio
+                Layout.bottomMargin: 3 * scaleRatio
+                Layout.fillWidth: true
+                color: MoneroComponents.Style.dividerColor
+                opacity: MoneroComponents.Style.dividerOpacity
+            }
+
+            WizardMenuItem {
+                headerText: qsTr("Open a wallet from file") + translationManager.emptyString
+                bodyText: qsTr("Import an existing .keys wallet file from your computer.") + translationManager.emptyString
+                imageIcon: "../images/open-wallet-from-file.png"
+
+                onMenuClicked: {
+                    wizardStateView.state = "wizardOpenWallet1"
+                }
+            }
+
+            Rectangle {
+                Layout.preferredHeight: 1
+                Layout.topMargin: 3 * scaleRatio
+                Layout.bottomMargin: 3 * scaleRatio
+                Layout.fillWidth: true
+                color: MoneroComponents.Style.dividerColor
+                opacity: MoneroComponents.Style.dividerOpacity
+            }
+
+            WizardMenuItem {
+                headerText: qsTr("Restore wallet from keys or mnemonic seed") + translationManager.emptyString
+                bodyText: qsTr("Enter your private keys or 25-word mnemonic seed to restore your wallet.") + translationManager.emptyString
+                imageIcon: "../images/restore-wallet.png"
+
+                onMenuClicked: {
+                    wizardController.restart();
+                    wizardController.createWallet();
+                    wizardStateView.state = "wizardRestoreWallet1"
+                }
+            }
+
+            RowLayout {
```

Comment:
Perhaps these two buttons would look better centered.

Reply:
Don't think it will look OK since it is a change from the 'far-left' & 'far-right' button positions we have currently. In addition, there is no guarantee that there are 2 buttons (depending on custom decorations).

Reply:
Feel free to PR for it after this is merged though if you think it's important.